pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -205,6 +208,32 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "Clean pieces stored using ttl",
+				Version:     4,
```

Comment:
Please change it into version 5.

Release-alpha4 contains a version 4 migration already and would not execute your code.

Reply:
done